build/asset-manifest.json

```diff
@@ -1,11 +1,11 @@
 {
   "files": {
-    "main.css": "/static/css/main.392d2585.css",
-    "main.js": "/static/js/main.d18a8be1.js",
+    "main.css": "/static/css/main.b8b807c2.css",
```

Comment:
can we remove the build folder from the checked in code?  it cause unnecessary complexities